Task 5/Array Helpers.js

```diff
@@ -0,0 +1,26 @@
+Array.prototype.square = function(){ return this.map( value => value*value ) };
+Array.prototype.cube = function(){ return this.map( value => value*value*value ) };
+Array.prototype.sum = function(){ 
+  let sum = 0;
+  this.forEach( value => sum += value );
+  return sum;
+};
+Array.prototype.average = function(){ 
+  let sum = 0, count = 0;
+  this.forEach( value => {
+    sum += value; 
+    count++; 
+  });
+  return sum/ count;
```

Comment:
You have calculated the `sum` in the above function `Array.prototype.sum`. This can be further simplified to 

```
Array.prototype.average = function() {
  return this.sum() / this.length;
};
```